src/app/customer/getPasswordRequirements.ts

```diff
@@ -0,0 +1,42 @@
+import { CustomerPasswordRequirements, ShopperConfig } from '@bigcommerce/checkout-sdk';
+
+export interface PasswordRequirements {
+    minLength: number;
+    alpha: RegExp;
+    numeric: RegExp;
+    description?: string;
+}
+
+export function getPasswordRequirementsFromConfig(config: ShopperConfig): PasswordRequirements {
+    const {
+        passwordRequirements: {
+            minlength,
+            error: description,
+            alpha,
+            numeric,
+        },
+    } = config;
+
+    return getPasswordRequirements({
+        minlength,
+        description,
+        alpha,
+        numeric,
+    });
+}
+
+export default function getPasswordRequirements({
+    minlength,
+    description,
+    alpha,
+    numeric,
+}: CustomerPasswordRequirements): PasswordRequirements {
+    const allSlashes = new RegExp('/', 'g');
+
+    return {
+        minLength: minlength,
+        description,
+        alpha: new RegExp(alpha.replace(allSlashes, '')),
```

Comment:
Do you just want to remove the start and end of `/`, or all occurrences of `/`?

Reply:
I think so. I think this is another copy-pasted code from ng-checkout https://github.com/bigcommerce-labs/ng-checkout/blob/master/src/app/directives/customer-password/customer-password.controller.js#L49